libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.SendActivity.schema

```diff
@@ -1,14 +1,14 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "Send Activity Action",
-    "description": "This is a action which sends an activity to the user",
+    "title": "Send an activity",
+    "description": "Respond with an activity.",
     "type": "object",
     "properties": {
         "activity": {
             "$type": "Microsoft.IActivityTemplate",
-            "title": "Activity",
-            "description": "Activity to send to the user"
+            "title": "Response",
```

Comment:
If we do this we should rename the object in the SDK.

Reply:
updated.